README.md

```diff
@@ -145,6 +145,8 @@ repositories:
  - mozilla/telemetry-dashboard: ['mentored', 'good first issue']
 ```
 
+For the `/languages/:language` route, we get issues tagged with labels in README from repositories whose primary language is the language or those labelled `lang=[language]`.
```

Comment:
> we get issues tagged with labels in README

I'm not sure I understand the link with the `README`. Could you help me understand, please?

Reply:
🙈aah my bad! I mean markdown file 🙈